### PR TITLE
feat: [OCA-1273]  - Add February 2024 Marketplace apps: NATS + Jupyter

### PR DESCRIPTION
**Pull request by tbaka**
## Description 📝
Contains assets and entries for new OCAs, NATS Single Node and JupyterLab Notebook.

## Changes  🔄
List any change relevant to the reviewer.
updates oneClickApps.ts

### CODE REVIEW

**Review by mjac0bs — COMMENTED** [review]
@tbaka Thanks for the additions. There are a few typos to correct in the code, one of which is causing a typecheck failure. We'll also want to update `marketplace.json` and add a changeset (via `yarn changeset`) for our changelog.

File: packages/manager/src/features/OneClickApps/oneClickApps.ts [hunk 1]
```diff
@@ -1298,6 +1319,28 @@ export const oneClickApps: OCA[] = [
...
+  {
+    alt_description:
+      `Microservice centeric stream processing.`,
+    alt_name: 'Microservice messaging bus',
+    categories: ['Development'],
+    colors: {
+      end: '000000',
+      start: '0086FF',
+    },
+    description: 'NATS is an distributed PubSub technology that enables applications to securely communicate across any combination of cloud vendors, on-premise, edge, web and mobile, and devices.',
```
Comment (carrillo-erik) [inline packages/manager/src/features/OneClickApps/oneClickApps.ts:1331]: I think the `n` is unnecessary here, should read as follows: `NATS is a distributed....`

**Comment by carrillo-erik** [comment]
Please include a `changeset`, refer to step 7 on the following doc: https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md

**Review by carrillo-erik — APPROVED** [review]

File: packages/manager/public/assets/white/juptyer.svg
Comment (jdamore-linode) [inline packages/manager/public/assets/white/juptyer.svg:1]: Can we rename this to `jupyter.svg`?

@tbaka I'm also seeing an E2E test failure related to this -- on closer inspection, it seems to be caused by a similar misspelling in the APIv4 StackScripts response for Jupyter:

Screenshot 2024-02-09 at 12 17 23 PM

Is there any way to fix that?

Comment (tbaka) [reply packages/manager/public/assets/white/juptyer.svg:1]: I've corrected the typo in the SVG, and in the Admin Stackscripts. That should resolve the E2E tests.

Comment (jdamore-linode) [reply packages/manager/public/assets/white/juptyer.svg:1]: Thanks @tbaka!

**Review by jdamore-linode — APPROVED** [review]

**Review by hana-linode — APPROVED** [review]

**Pull request merged**